osquery/tables/applications/posix/docker.cpp

```diff
@@ -8,99 +8,17 @@
  *
  */
 
-#include <cstdlib>
-#include <cstring>
-#include <iostream>
-
-#include <boost/algorithm/string/classification.hpp>
-#include <boost/algorithm/string/predicate.hpp>
-#include <boost/algorithm/string/split.hpp>
-#include <boost/asio.hpp>
+#include <boost/algorithm/string.hpp>
 #include <boost/foreach.hpp>
 
-#if !defined(BOOST_ASIO_HAS_LOCAL_SOCKETS)
-#error Boost error: Local sockets not available
-#endif
-
-#include <osquery/flags.h>
 #include <osquery/logger.h>
 #include <osquery/tables.h>
-
-#include "osquery/core/json.h"
-
-namespace pt = boost::property_tree;
-namespace local = boost::asio::local;
+#include <osquery/tables/applications/posix/docker_api.h>
```

Comment:
nit: quotes and in it's own block.

Reply:
@diptanu, ping on this!

This include should be a new include block and use " " for the local search path.